src/DealClient.sol

```diff
@@ -28,32 +19,32 @@ contract DealClient {
         owner = msg.sender;
     }
 
-    function addCID(bytes calldata cidraw, uint size) public {
+    function addCID(bytes calldata pieceCID, uint size) public {
        require(msg.sender == owner);
-       cidSet[cidraw] = true;
-       cidSizes[cidraw] = size;
+       cidSet[pieceCID] = true;
+       //cidSizes[pieceCID] = size;
```

Comment:
As long as you agree it won't confuse users or break the tools we have already, please remove the size parameter from the addCID call signature entirely.  FYI @shrenujbansal I added this for security reasons but Im pretty sure I was wrong and this is unnecessary.

Reply:
And delete this instead of commenting out